code/src/components/Footer.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import technigoLogo from '../assets/logo.svg';
+import { FooterContainer, FooterText, FooterImage, Links } from './StyledComponents/FooterStyling';
+
+
+export const Footer = () => {
+  return (
+      <FooterContainer>
+        <FooterText>© 2020 Sofia Vaz Sousa</FooterText>
+        <FooterText>
+          <Links href="https://www.linkedin.com/in/sofiavazsousa/" target="_blank" rel="noopener noreferrer">LinkedIn</Links>
+              |
+          <Links href="https://github.com/sofiavazs" target="_blank" rel="noopener noreferrer">GitHub</Links>
+        </FooterText>
+        <FooterImage src={technigoLogo} alt="technigo logo" />
+        <FooterText>Project developed during the Technigo frontend development bootcamp</FooterText>
+      </FooterContainer>
+  );
+};
```

Comment:
Food for thought - in your mobile version the footer is "blocking" the clear all button. I do not know how to fix it but maybe that is something worth looking into again? :)

Reply:
Thanks! Yeah it was needed to scroll down and then it would be okay, but I spent hours trying to fix it and decided to just hide it on mobile version, because Safari hates me 😓